botcore/_bot.py

```diff
@@ -49,9 +50,10 @@ def __init__(
             guild_id: The ID of the guild use for :func:`wait_until_guild_available`.
             allowed_roles: A list of role IDs that the bot is allowed to mention.
             http_session (aiohttp.ClientSession): The session to use for the bot.
-            redis_session: The
-                ``[async_rediscache.RedisSession](https://github.com/SebastiaanZ/async-rediscache#creating-a-redissession)``
-                to use for the bot.
+            redis_session: The `async_rediscache.RedisSession`_ to use for the bot.
+            api_client: The api_client instance to use for the bot.
```

Comment:
Shouldn't it be something like this (not sure how exactly sphinx wants it)?
```suggestion
            api_client: The `APIClient` instance to use for the bot.
```

Reply:
Changed in forcepush